schema/spicedb_schema.yaml

```diff
@@ -0,0 +1,61 @@
+schema: |-
```

Comment:
The file name can be generic like bootstrap yaml, since it will have schema, relations.

Reply:
I see your point. I'm wondering though if bootstrap is too vague for a person not familiar with (or who has forgotten) what is going on. Might be nice to indicate what kind of data spicedb is getting bootstrapped with.